dags/js/tasks/convert-tiff-images-in-expanded-bucket-to-jpeg-images.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const sharp = require('sharp');
+const uuidv4 = require('uuid/v4');
+const getS3Client = require('../aws/get-s3-client');
+
+async function convertTiffImagesInExpandedBucketToJpegImages() {
+
+  function deleteFile(fileName) {
+    fs.unlinkSync(fileName);
```

Comment:
to be discussed: use of sync IO in production code

Reply:
I can update this.

Reply:
updated to async version.